src/items/itemDialog.component.tsx

```diff
@@ -750,33 +790,108 @@ function ItemDialog(props: ItemDialogProps) {
-        <Box
-          sx={{ display: 'flex', alignItems: 'center', width: '100%' }}
-        ></Box>
-        <Box
-          sx={{
-            display: 'flex',
-            justifyContent: 'center',
-            width: '100%',
-            my: 2,
-          }}
+        );
+      case 2:
+        return (
+          <Grid item xs={12}>
+            <SystemsTableView
+              systemsData={systemsData}
+              systemsDataLoading={systemsDataLoading}
+              onChangeParentId={setParentSystemId}
+              // Use most unrestricted variant (i.e. copy with no selection)
+              selectedSystems={[]}
+              type="copyTo"
+            />
+          </Grid>
+        );
+    }
+  };
+
+  return (
+    <Dialog
+      open={open}
+      onClose={handleClose}
+      maxWidth="lg"
+      PaperProps={{ sx: { height: '705px' } }}
+      fullWidth
+    >
+      <DialogTitle>
+        <Grid item xs={12}>{`${type === 'edit' ? 'Edit' : 'Add'} Item`}</Grid>
+      </DialogTitle>
+      <DialogContent>
+        <Stepper
+          nonLinear
+          activeStep={activeStep}
+          orientation="horizontal"
+          sx={{ marginTop: 2 }}
         >
+          {STEPS.map((label, index) => (
+            <Step key={label}>
+              <StepButton onClick={() => setActiveStep(index)}>
+                {label}
+              </StepButton>
+            </Step>
+          ))}
+        </Stepper>
+        {activeStep === 2 && (
+          <Grid item xs={12}>
+            <Breadcrumbs
+              breadcrumbsInfo={parentSystemBreadcrumbs}
+              onChangeNode={setParentSystemId}
+              onChangeNavigateHome={() => {
+                setParentSystemId(null);
+              }}
+              navigateHomeAriaLabel={'navigate to systems home'}
+            />
+          </Grid>
+        )}
+        <Box sx={{ marginTop: 2 }}>{renderStepContent(activeStep)}</Box>
+      </DialogContent>
+      <DialogActions>
+        <Button onClick={handleClose} sx={{ mr: 'auto' }}>
+          Cancel
+        </Button>
+        <Button disabled={activeStep === 0} onClick={handleBack} sx={{ mr: 2 }}>
+          Back
+        </Button>
+
+        {activeStep === STEPS.length - 1 ? (
           <Button
-            variant="outlined"
-            sx={{ width: '50%', mx: 1 }}
-            onClick={handleClose}
+            disabled={
+              !itemDetails.system_id ||
+              catchAllError ||
+              formErrorMessage !== undefined ||
+              propertyErrors.some((value) => value === true) ||
+              (!!itemDetails.warranty_end_date &&
+                !isValidDateTime(itemDetails.warranty_end_date)) ||
+              (!!itemDetails.delivered_date &&
+                !isValidDateTime(itemDetails.delivered_date))
+            }
+            onClick={type === 'edit' ? handleEditItem : handleAddItem}
+            sx={{ mr: 3 }}
           >
-            Cancel
+            Finish
```

Comment:
```suggestion
            Place here
```
Could this work? I agree it is confusing when it says select system since we don't use checkboxes and rely on placing them into the system.

Reply:
would it be better to change the label because "Place here" doesn't suggest that it going to send a post request.

Reply:
"Place item into a system" instead  of "Select a system"